### PR TITLE
[eclipse/xtext#1633] Fix broken classpath files after m2e upgrade via oomph

### PR DESCRIPTION
**Pull request by szarnekow**
This is an alternative fix to #1301 while the changes are not yet available in m2e-tycho

Running the setup tasks after an m2e upgrade classpath will attempt to replace target/classes with bin output folders again.

### CODE REVIEW

**Comment by cdietrich** [comment]
move to 2.24.
also needs rebase

**Comment by szarnekow** [comment]
Rebased. What do you think, @cdietrich ?

File: org.eclipse.xtext.builder/.classpath [hunk 1]
```diff
@@ -2,12 +2,8 @@
...
 	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
-	<classpathentry kind="src" path="src"/>
-	<classpathentry kind="src" path="emf-gen">
-		<attributes>
-			<attribute name="ignore_optional_problems" value="true"/>
-		</attributes>
-	</classpathentry>
-	<classpathentry kind="src" path="deprecated"/>
+	<classpathentry kind="src" path="src/"/>
+	<classpathentry kind="src" path="emf-gen"><attributes><attribute name="ignore_optional_problems" value="true"/></attributes></classpathentry>
```
Comment (cdietrich) [inline org.eclipse.xtext.builder/.classpath:6]: formatting change intended?

Comment (szarnekow) [reply org.eclipse.xtext.builder/.classpath]: I think I fixed it.

File: org.eclipse.xtext.builder/.classpath [hunk 2]
```diff
@@ -2,12 +2,12 @@
...
 	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
-	<classpathentry kind="src" path="src"/>
+	<classpathentry kind="src" path="src/"/>
 	<classpathentry kind="src" path="emf-gen">
 		<attributes>
 			<attribute name="ignore_optional_problems" value="true"/>
 		</attributes>
 	</classpathentry>
-	<classpathentry kind="src" path="deprecated"/>
+	<classpathentry kind="src" path="deprecated/"/>
```
Comment (szarnekow) [inline org.eclipse.xtext.builder/.classpath:11]: Hmm I've no idea why m2e used a trailing slash on the source paths, but it seems to be fine.

**Review by cdietrich — APPROVED** [review]

**Pull request merged**